src/jobs/build.yml

```diff
@@ -1,9 +1,12 @@
 description: >
   Start building an Android project on CircleCI
 
-executor:
-  name: android
-  sdk-version: "23"
+parameters:
```

Comment:
I'd leave this decision to the Orbs side of the house but my vote is no.

1. I think it's best for users to make an educated decision on which image they use.
2. No matter what we choose, it will eventually become older and people will request we change it, will automatically forces a major orb update.